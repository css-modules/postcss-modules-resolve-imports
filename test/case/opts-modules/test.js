'use strict';

const {resolve} = require('path');
const resolveImports = require('../../../index');
const setup = require('../../setup');

test('opts-modules', () => {
  const {resulting, exports: tokens} = setup(
    'local-by-default',
    'extract-imports',
    'scope',
    resolveImports({
      resolve: {
        modules: [resolve(__dirname, 'lib')],
      },
    })
  )(__dirname);

  expect(resulting).toMatchSnapshot();
  expect(tokens).toMatchSnapshot();
});
