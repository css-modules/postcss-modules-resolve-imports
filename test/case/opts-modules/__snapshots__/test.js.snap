// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opts-modules 1`] = `
"/* imported from lib/button/index.css */._index_button
{
  /*common button styles*/
}/* imported from source.css */._source_continueButton
{
  color: green;
}
"
`;

exports[`opts-modules 2`] = `
Object {
  "continueButton": "_source_continueButton _index_button",
}
`;
